src/vivarium/framework/state_machine.py

```diff
@@ -108,45 +110,46 @@ class Transition:
 
     Parameters
     ----------
-    input_state: State
```

Comment:
In line 97, shouldn't `_process_trigger()` have both input and output types specified?